qa/tasks/cephfs/test_admin.py

```diff
@@ -1440,7 +1440,7 @@ def test_single_path_rootsquash(self):
         self.captester.conduct_neg_test_for_chown_caps()
         self.captester.conduct_neg_test_for_truncate_caps()
 
-    def test_multifs_single_path_rootsquash(self):
+    def _test_multifs_single_path_rootsquash(self):
```

Comment:
```suggestion
    @unittest.skip("explanation")
    def test_multifs_single_path_rootsquash(self):
```

Reply:
Much better, thanks!! :)

Reply:
Switched to `self.skipTest()` since it is already available unlike `unittest.skip()`